UPGRADE-5.0.md

```diff
@@ -5,6 +5,7 @@
 - The calendar feature was removed. See https://github.com/mautic/mautic/pull/11270
 - The Froala assets are disabled by default. Enable them if you use the legacy email or page builder. See https://github.com/mautic/mautic/pull/12416
 - New shortening service in Mautic 5, reconfiguration is needed after migrating to Mautic 5. A user reads the documentation to understand how to set it up in the configuration.
+- Removed Show after X submissions option from forms. See documentation for the setup real progressive profiling for forms.
```

Comment:
Can you please add link to that documentation? Because if the link was here I wouldn't be so confused.

Reply:
This PR should also deprecated all the code that is using this feature. Like deprecating the getter, setter and the property in https://github.com/mautic/mautic/blob/5.x/app/bundles/FormBundle/Entity/Field.php#L791-L799

I bet if we won't do this then this dead code will be present in Mautic 10

Reply:
Or perhaps remove the code completely?